site/index.html

```diff
@@ -501,6 +501,37 @@ <h4 id="convolver">Convolver</h4>
                     <button id="play-drums" class="play action"></button>
                 </div>
 
+            <a name="tremolo"></a><h4>Tremolo</h4>
```

Comment:
I added a sidebar in the public site with links to each of the effects for easier navigation (```div``` element of class ```menu```), at around line 21 of this file. It would be good to add Tremolo to that menu.